### ISSUE TITLE
compatibility with emacs-prelude

### ISSUE THREAD

**Issue by nha** [editorconfig/editorconfig-emacs#37]
Hello,

This is more a question than an issue I guess.

I have been using `emacs` and `editorconfig-emacs` for a little bit of time, and I recently switched to `emacs-prelude` (https://github.com/bbatsov/prelude).

But I do not manage to make it work with this setup. I am unsure if I do something wrong, or if this is not compatible with `emacs-prelude` ?

Is there a way to check that the .editorconfig file is loaded from emacs ?

I tried two different ways : 

```
(prelude-require-packages '( editorconfig ))
(load "editorconfig")
````
and 

```
(add-to-list 'load-path "~/.emacs.d/personal")
(load "editorconfig")
````

editorconfig is installed (probably via `brew` but I can't remember. It did work on this machine at some point though).

```
editorconfig -v
EditorConfig C Core Version 0.12.0
```

Thanks for making this :)

**Comment by xuhdev** [comment]
Currently there is no way, but you can definitely modify `.editorconfig` and see whether things change in prelude. Let me know if you still have questions :)

**Comment by nha** [comment]
Thanks for your answer :)

Well I don't get something noticeably different when changing my `.editorconfig`, hence my question. I can check that `editorconfig path-to-file` gives me the correct settings though.

Any idea what would be the first step to debug my configuration ?

**Comment by xuhdev** [comment]
@nha You can try changing `indent_size`, which should lead to different size of indentation in your file. You can also check the variable `edconf-find-file-hook`.

**Comment by nha** [comment]
@xuhdev I tried changing this setting (as well as the maximum line size). I'm quite sure I'm doing something wrong here, but I can't figure out what.

`M-x describe-variable` edconf-find-file-hook
But it doesn't exist ? I tried instead :
`M-x describe-function` edconf-find-file-hook`

```
edconf-find-file-hook is a compiled Lisp function in
`editorconfig.el'.

(edconf-find-file-hook)

Not documented.
````

Not sure if that's what you wanted ?
You guessed it I'm an emacs beginner...
Sorry for taking some of your time on this I'll dig a bit more.

Thanks again :)
